addons/web_tour/static/src/js/tip.js

```diff
@@ -385,8 +385,12 @@ var Tip = Widget.extend({
     },
     _unbind_anchor_events: function () {
         this.$anchor.off(".anchor");
-        this.$consumeEventAnchors.off(".anchor");
-        this.$scrolableElement.off('.Tip');
+        if (this.$consumeEventAnchors) {
+            this.$consumeEventAnchors.off(".anchor");
+        }
+        if (this.$scrolableElement) {
+            this.$scrolableElement.off('.Tip');
+        }
```

Comment:
Why is this necessary? You found cases where the tip are destroyed while not fully initialized?